samples/nhibernate/simple/sample.md

```diff
@@ -0,0 +1,89 @@
+---
+title: NHibernate persistence Sample
+summary: Using NHibernate to store Sagas and Timeouts.
+reviewed: 2016-08-29
+component: NHibernate
+tags:
+ - Saga
+ - Timeout
+related:
+ - nservicebus/sagas
+ - nservicebus/persistence
+---
+
+
+## Code walk-through
+
+This sample shows a simple Client + Server scenario.
+
+ * `Client` sends a `StartOrder` message to `Server`.
+ * `Server` starts an `OrderSaga`.
+ * `OrderSaga` requests a timeout with a `CompleteOrder` data.
+ * When the `CompleteOrder` timeout fires the `OrderSaga` publishes a `OrderCompleted` event.
+ * The Server then publishes a message that the client has subscribed to.
+ * `Client` handles `OrderCompleted` event.
+
+
+### NHibernate Config
+
+Configure NHibernate with the right driver, dialect and connection string. Since NHibernate needs a form of mapping of the class to the database table, the configuration code also does that with ModelMapper API. Finally, the configuration is used to run the endpoint.
+
+snippet:config
+
+
+### Order Saga Data
+Notice that due to a bug, SagaData class needs to be inside a namespace. Also to use NHibernate's lazy-loading all the properties must be marked as `virtual`.
```

Comment:
hmmm I‌ see that the bug is closed, but the sample wasn't running because of the exact bug! I'll check again.